### ISSUE TITLE
Synchronization with VA Acronym Lookup

### ISSUE THREAD

**Issue by franklin-hines** [department-of-veterans-affairs/acronyms#55]
Not the highest priority, but has anyone looked into whether it would be feasible (and there would be enough value) to synchronize (or centralize) the acronym data in acronyms.csv with the data that is used by the VA Acronym Lookup (https://vaww.va.gov/acronyms/index.cfm)? I've noticed some omissions in the VA Acronym Lookup and have submitted suggestions to help improve the consistency between it and this tool, but hoping those add'l efforts can be avoided at some point for everyone.

**Comment by karlbrown-va** [comment]
The acronyms file has a 'Context' field that could serve as a sort of categorization, but it's not filled in completely.

@franklin-hines a while back I merged in a bunch of terms from another VA source. I think one could make the argument that this method (text-based, PR-editable) is the best way of maintaining such a list potentially..

**Comment by franklin-hines** [comment]
@stphnwlkr that is great to hear 👏 and @karlbrown-va 's suggestion to consider using 'Context' in the acronyms.csv seems feasible, though it wouldn't be a fixed list of categories and would have to populate the numerous null values that exist today as Karl also pointed out. 

Perhaps another consideration would be to extend the acronyms.csv to accommodate for a "Category" field (that the Slackbot could either incorporate and return to Slack users or ignore as part of an initial solution) and have some means to assert that a Category that is being added for an acronym aligns with a master list.

**Comment by karlbrown-va** [comment]
And now you've gone from maintaining a CSV file to building a master data management system :)

**Comment by franklin-hines** [comment]
@karlbrown-va yeah as soon as I suggested being able to check against a master list for categories, I was thinking referential integrity and building this out in a relational DB would ease those types of checks.

**Comment by stphnwlkr** [comment]
Just a quick update. They have decided to move the official VA list to Sharepoint which means it won't be publicly available, but will be available to staff on GFE devices without being on the network --- if enrolled in MS365 Direct which is currently being rolled out. They have this list as well as the original and will be adding categorization. The plan is to go with broad categories (e.g., internal organization, information technology, or healthcare) as well as offices of primary responsibility (e.g., Veterans Health Administration, Office of Information and Technology) this combination should help users when they encounter duplicate abbreviations.
